docker/Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN chown -R actions ~actions && /home/actions/actions-runner/bin/installdepende
 RUN apt-get update && apt-get install -y udev usbutils
 RUN mkdir -p /etc/udev/rules.d/
 COPY 49-stlinkv3.rules /etc/udev/rules.d/49-stlinkv3.rules
-RUN service udev restart || 
+RUN service udev restart
```

Comment:
To my understanding the `||` should proceed with the Dockerfile build even if the restart fails. I guess it's okay to fail the build.

Reply:
In my case the Dockerfile build failed repeatedly due to this... It's just a bash OR, and shouldn't do anything unless `service udev restart` returns non-0.  I'm also no bash expert, but after removing it the Dockerfile build ran through without issue.

Reply:
Oh, interesting. Thanks, good to know.